ts/features/idpay/initiative/details/components/RefundDetailsComponent.tsx

```diff
@@ -0,0 +1,173 @@
+  const initiativeId = useIOSelector(idpayInitiativeIdSelector);
+
+  const isRejected = refund.operationType === OperationTypeEnum.REJECTED_REFUND;
+
+  const handleEditIbanPress = () => {
+    closeBottomSheet();
+
+    // The utilization of the NavigationService is imperative, as the library employed for presenting bottom sheets
+    // utilizes its own NavigationContainer. Consequently, utilizing the "useNavigation" hook in this scenario
+    // would result in an error.
+    NavigationService.dispatchNavigationAction(
+      CommonActions.navigate(
+        IDPayConfigurationRoutes.IDPAY_CONFIGURATION_MAIN,
+        {
+          screen: IDPayConfigurationRoutes.IDPAY_CONFIGURATION_IBAN_ENROLLMENT,
+          params: {
+            initiativeId
+          }
+        }
+      )
+    );
+  };
+
+  return (
+    <>
+      {isRejected && (
+        <>
+          <VSpacer size={16} />
+          <InfoBox iconColor={IOColors.red}>
+            <Body>
+              {I18n.t("idpay.initiative.operationDetails.rejectedAdvice.text")}
+            </Body>
+            <VSpacer size={8} />
+            <Link color={"bluegreyDark"} onPress={handleEditIbanPress}>
+              {I18n.t(
+                "idpay.initiative.operationDetails.rejectedAdvice.editIban"
+              )}
+            </Link>
+          </InfoBox>
+          <VSpacer size={16} />
+        </>
+      )}
+      <View style={styles.detailRow}>
+        <Body>{I18n.t("idpay.initiative.operationDetails.iban")}</Body>
+        <Body weight="SemiBold">{/* TODO add iban */}</Body>
+      </View>
+      <View style={styles.detailRow}>
+        <Body>{I18n.t("idpay.initiative.operationDetails.refundAmount")}</Body>
+        <Body weight="SemiBold">
+          {formatNumberAmount(refund.amount || 0, true)}
+        </Body>
+      </View>
+      <View style={styles.detailRow}>
+        <Body>{I18n.t("idpay.initiative.operationDetails.resultLabel")}</Body>
+        <ResultLabel type={refund.operationType} />
+      </View>
+      <View style={styles.detailRow}>
+        <Body>Periodo di riferimento</Body>
+        <Body weight="SemiBold">
+          {`${format(refund.operationDate, "DD/MM/YY")} - ${format(
+            refund.operationDate,
+            "DD/MM/YY"
+          )}`}
+        </Body>
+      </View>
+      <View style={styles.detailRow}>
+        <Body>Data rimborso</Body>
+        <Body weight="SemiBold">
+          {format(refund.operationDate, "DD MMM YYYY, HH:mm")}
+        </Body>
+      </View>
+      <View style={styles.detailRow}>
+        <Body>TRN</Body>
+        <HSpacer size={16} />
+        <View style={[IOStyles.flex, IOStyles.row]}>
+          <Body
+            weight="SemiBold"
+            numberOfLines={1}
+            ellipsizeMode="tail"
+            style={IOStyles.flex}
+          >
+            {/* TODO add TRN */}
+          </Body>
+          <HSpacer size={8} />
+          <CopyButtonComponent textToCopy={/* TODO add TRN */ ""} />
```

Comment:
this probably needs a custom component to follow design (copy button -> copy icon in clickable ListItem)

Reply:
@dmnplb we should add this new component to the DS (https://www.figma.com/file/SuYjyjZlieQqRS8n1B6qbF/Flussi-App-ID-Pay?node-id=983:24951&t=tP6vxJxdGRyYAnRs-0)

Reply:
As per @dmnplb, using the default copy button is correct in this case.